<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyHomepage</class>
 <widget class="QWidget" name="MyHomepage">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_myhomepage">
     <layout class="QVBoxLayout" name="whole" stretch="0,22,3">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="return_button">
        <property name="text">
         <string>...</string>
        </property>
        <property name="arrowType">
         <enum>Qt::LeftArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="main" stretch="1,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="personal_info" stretch="3,4">
          <property name="spacing">
           <number>8</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="profile" stretch="2,1">
            <property name="spacing">
             <number>5</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="profile_photo" stretch="2,2,9,1,1,3">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="my_image">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="photo_edit">
                <property name="maximumSize">
                 <size>
                  <width>147</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>üñãÔ∏è</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="empty">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="nickname" stretch="1,9,1,1,1">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="styleSheet">
                 <string notr="true">font: 9pt &quot;ÂçéÊñá‰∏≠ÂÆã&quot;;</string>
                </property>
                <property name="text">
                 <string>  ÊòµÁß∞Ôºö</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="nickname_content">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="empty_3">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="nickname_edit">
                <property name="text">
                 <string>üñãÔ∏è</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="basic_info">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="real_name" stretch="1,9,1,1,1">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="styleSheet">
                 <string notr="true">font: 9pt &quot;ÂçéÊñá‰∏≠ÂÆã&quot;;</string>
                </property>
                <property name="text">
                 <string>  ÂßìÂêçÔºö</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="real_name_content">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="empty_5">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="real_name_edit">
                <property name="text">
                 <string>üñãÔ∏è</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="grade" stretch="1,9,1,1,1">
              <item>
               <widget class="QLabel" name="label_11">
                <property name="styleSheet">
                 <string notr="true">font: 9pt &quot;ÂçéÊñá‰∏≠ÂÆã&quot;;</string>
                </property>
                <property name="text">
                 <string>  Âπ¥Á∫ßÔºö</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="grade_content">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="empty_7">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="grade_edit">
                <property name="text">
                 <string>üñãÔ∏è</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="major" stretch="1,9,1,1,1">
              <item>
               <widget class="QLabel" name="label_12">
                <property name="styleSheet">
                 <string notr="true">font: 9pt &quot;ÂçéÊñá‰∏≠ÂÆã&quot;;</string>
                </property>
                <property name="text">
                 <string>  ‰∏ì‰∏öÔºö</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="major_content">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="empty_9">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="major_edit">
                <property name="text">
                 <string>üñãÔ∏è</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="birthdate" stretch="1,9,3">
              <item>
               <widget class="QLabel" name="label_13">
                <property name="styleSheet">
                 <string notr="true">font: 9pt &quot;ÂçéÊñá‰∏≠ÂÆã&quot;;</string>
                </property>
                <property name="text">
                 <string>  ÁîüÊó•Ôºö</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="dateEdit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="focusPolicy">
                 <enum>Qt::NoFocus</enum>
                </property>
                <property name="calendarPopup">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="movies" stretch="12,7">
          <property name="spacing">
           <number>15</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="watched_list" stretch="7,16">
            <property name="spacing">
             <number>5</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="title" stretch="8,1,1">
              <item>
               <widget class="QLabel" name="title_watched_list">
                <property name="font">
                 <font>
                  <family>ÂçéÊñá‰∏≠ÂÆã</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 10pt &quot;ÂçéÊñá‰∏≠ÂÆã&quot;;</string>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;ÈÇ£‰∫õÂπ¥ÔºåÊàëÂú®Âú∞Èô¢ÁúãËøáÁöÑÁîµÂΩ±......&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="movie_list">
              <item row="1" column="4">
               <widget class="QPushButton" name="movie10">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="movie2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="movie7">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QPushButton" name="movie8">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QPushButton" name="movie5">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QPushButton" name="movie3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QPushButton" name="movie4">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QPushButton" name="movie9">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QPushButton" name="movie1">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="movie6">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="top_5" stretch="3,4">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="title_2" stretch="8,1,1">
              <item>
               <widget class="QLabel" name="title_top_5">
                <property name="font">
                 <font>
                  <family>ÂçéÊñá‰∏≠ÂÆã</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 10pt &quot;ÂçéÊñá‰∏≠ÂÆã&quot;;</string>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Êó†Ê≥ïÂâ≤ËàçÁöÑÂøÉÂ§¥Áà±‚Äî‚ÄîÊàëÁöÑTOP 5&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="top_list" columnstretch="0,0,0,0,0">
              <property name="spacing">
               <number>4</number>
              </property>
              <item row="0" column="4">
               <widget class="QPushButton" name="top5">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QPushButton" name="top4">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QPushButton" name="top3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QPushButton" name="top1">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="top2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="switch_button">
        <item alignment="Qt::AlignBottom">
         <widget class="QPushButton" name="publicSpaceButton">
          <property name="font">
           <font>
            <family>ÂçéÊñá‰∏≠ÂÆã</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>È¶ñÈ°µ</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignBottom">
         <widget class="QPushButton" name="me">
          <property name="font">
           <font>
            <family>ÂçéÊñá‰∏≠ÂÆã</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Êàë</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
